server/config/brain/light/answers.fr.json

```diff
@@ -14,5 +14,9 @@
   {
     "label": "light.turn-off.fail",
     "answers": ["Je n'ai pas réussi à éteindre la lumière."]
+  },
+  {
+    "label": "light.not-found",
+    "answers": ["Je n'ai pas trouvé de lumière dans cette pièce."]
```

Comment:
Isn't there an "S" after lumière? ( Je n'ai pas trouvé de lumièreS ?) I'm not sure :D

Reply:
From this page, both are ok https://www.francaisfacile.com/exercices/exercice-francais-2/exercice-francais-13742.php
If you prefer with an S, I can add it.

Reply:
It's ok like this then. Just checking :)